migration.go

```diff
@@ -129,21 +129,23 @@ func (m *Migration) Buffer() error {
 
 	// start reading from body, peek won't move the read pointer though
 	// poor man's solution?
-	if _, err := b.Peek(int(m.BufferSize)); err != nil && err != io.EOF {
+	// related issue : https://github.com/gobuffalo/packd/issues/9
+	migrFile, err := b.Peek(int(m.BufferSize))
+	if err != nil && err != io.EOF {
 		return err
 	}
 
 	m.FinishedBuffering = time.Now()
 
 	// write to bufferWriter, this will block until
 	// something starts reading from m.Buffer
-	n, err := b.WriteTo(m.bufferWriter)
+	n, err := m.bufferWriter.Write(migrFile)
```

Comment:
I don't think you can replace [`WriteTo()`](https://golang.org/pkg/bufio/#Reader.WriteTo) this way since it exhausts the underlying reader. My understanding is that `Peek()` is just used to initially populate the buffer.